SECURITY.md

```diff
@@ -0,0 +1,43 @@
+# Security Policy
+
+This policy applies to **all repositories under the [github.com/yeoman](https://github.com/yeoman) organization**.
+
+## Reporting a Vulnerability
+
+We take security vulnerabilities seriously and appreciate your efforts to responsibly disclose any issues. Please follow the guidelines below to report any security issues **privately**.
+
+### Primary Reporting Method: GitHub Security Advisory
+
+If you discover a security vulnerability, it is crucial that you **do not create a public issue** under any circumstances. Public issues can inadvertently expose the vulnerability, potentially leading to exploitation before a fix is available.
+
+Instead, please report the vulnerability via the [GitHub Security Advisory](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability) for the relevant repository. This private channel ensures only maintainers can access the details, enabling a timely and secure resolution. We fully utilize the capabilities that the GitHub Security Advisory platform provides, including private communication channels and coordinated disclosure options.
+
+### Secondary Reporting Option: Email
+
+If you are unable to use the GitHub Security Advisory for any reason, you may report the issue via email to `ulises@linux.com`. If you would like to encrypt sensitive information, please use our PGP key, available at: [https://github.com/ulisesgascon.gpg](https://github.com/ulisesgascon.gpg)
```

Comment:
@JoshuaKGoldberg this is a workaround until we have an official team email for this :+1: